Gems/Atom/RHI/Code/Include/Atom/RHI/ShaderResourceGroupDebug.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ * 
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+namespace AZ
+{
+    namespace RHI
+    {
+        class ConstantsData;
+        struct DrawItem;
+        class ShaderResourceGroup;
+
+        void PrintConstantDataDiff(const ShaderResourceGroup& shaderResourceGroup, ConstantsData& referenceData, bool updateReferenceData = false);
```

Comment:
Feel like there should be a short comment here explaining these functions.

Reply:
Will add